opslevel/resource_opslevel_integration_google_cloud.go

```diff
@@ -0,0 +1,252 @@
+		resourceModel.TagsOverrideOwnership = types.BoolNull()
+	} else {
+		resourceModel.TagsOverrideOwnership = types.BoolValue(googleCloudIntegration.GoogleCloudIntegrationFragment.TagsOverrideOwnership)
+	}
+	resourceModel.Projects = OptionalStringListValue(flattenProjectsArray(googleCloudIntegration.GoogleCloudIntegrationFragment.Projects))
+
+	return resourceModel
+}
+
+func (r *integrationGoogleCloudResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_integration_google_cloud"
+}
+
+func (r *integrationGoogleCloudResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "Google Cloud Integration resource",
+
+		Attributes: map[string]schema.Attribute{
+			"aliases": schema.ListAttribute{
+				Description: "All of the aliases attached to the integration.",
+				Computed:    true,
+				ElementType: types.StringType,
+			},
+			"client_email": schema.StringAttribute{
+				Description: "The service account email OpsLevel uses to access the Google Cloud account.",
+				Required:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"private_key": schema.StringAttribute{
+				Description: "The private key for the service account that OpsLevel uses to access the Google Cloud account.",
+				Required:    true,
+				Sensitive:   true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"created_at": schema.StringAttribute{
+				Description: "The time this integration was created.",
+				Computed:    true,
+			},
+			"id": schema.StringAttribute{
+				Description: "The ID of the integration.",
+				Computed:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"installed_at": schema.StringAttribute{
+				Description: "The time that this integration was successfully installed, if null, this indicates the integration was not completely installed.",
+				Computed:    true,
+			},
+			"name": schema.StringAttribute{
+				Description: "The name of the integration.",
+				Required:    true,
+			},
+			"ownership_tag_keys": schema.SetAttribute{
+				ElementType: types.StringType,
+				Description: "An Array of tag keys used to associate ownership from an integration. Max 5",
+				Optional:    true,
+				Validators: []validator.Set{
+					setvalidator.SizeBetween(1, 5),
+				},
+			},
+			"ownership_tag_overrides": schema.BoolAttribute{
+				Description: "Allow tags imported from Google Cloud to override ownership set in OpsLevel directly.",
+				Optional:    true,
+			},
+			"projects": schema.ListAttribute{
+				Description: "A list of the Google Cloud projects that were imported by the integration.",
+				Computed:    true,
+				ElementType: types.StringType,
```

Comment:
Why a list of just the `names` instead of an array of the full object our API returns?

Reply:
@rocktavious I originally tried to output the full object here in the first commit but was not able to get it to work. I'll find a way to fix this.

Reply:
Here is an example @davidbloss just did - https://github.com/OpsLevel/terraform-provider-opslevel/pull/429/files#diff-19450e4519df7a48a6e6dfd66513f43280e2b71226b6fa80406cc1c1f8200b51R78